libs/domain/checkout/shipping-method/shipping-method.styles.ts

```diff
@@ -18,10 +18,11 @@ export const styles = css`
     display: contents;
   }
 
-  .no-methods {
-    --oryx-icon-size: 40px;
```

Comment:
Why did you drop the icon size?

Reply:
Current icon on netlify appears to be the wrong size. Fixing it here, according to designs from the address list component story.

Reply:
Nevermind, Helen says it should be 24px. The size is correct.